server/src/ZetaSqlWrapper.ts

```diff
@@ -65,7 +65,12 @@ export class ZetaSqlWrapper {
   }
 
   getTableRef(model: ManifestModel, name: string): string[] | undefined {
-    return model.refs.find(ref => ref.indexOf(name) === ref.length - 1);
+    const tableRef = model.refs.find(ref => ref.indexOf(name) === ref.length - 1);
+    if (tableRef) {
+      return tableRef;
+    }
+    const aliasedModel = this.dbtRepository.models.find(m => m.alias === name);
+    return aliasedModel && model.refs.some(ref => ref.indexOf(aliasedModel.name) === ref.length - 1) ? [aliasedModel.name] : undefined;
```

Comment:
There is duplication here:
`model.refs.some(ref => ref.indexOf(aliasedModel.name) === ref.length - 1)`